libs/ledgerjs/packages/cryptoassets/src/data/trc20.json

```diff
@@ -1 +1,10 @@
-[["TAFjULxiVgT4qWk6UZwjqwZXTSaGaqnVp4","BTT","BitTorrent","TAFjULxiVgT4qWk6UZwjqwZXTSaGaqnVp4",18,null,null,null],["TCFLL5dx5ZJdKnWuesXxi1VPwjLVmWZZy9","JST","JUST GOV","TCFLL5dx5ZJdKnWuesXxi1VPwjLVmWZZy9",18,null,null,null],["TLa2f6VPqDgRE67v1736s7bJ8Ray5wYjU7","WIN","WINK","TLa2f6VPqDgRE67v1736s7bJ8Ray5wYjU7",6,null,null,null],["TMwFHYXLJaRUPeW6421aqXL4ZEzPRFGkGT","USDJ","JUST Stablecoin","TMwFHYXLJaRUPeW6421aqXL4ZEzPRFGkGT",18,null,null,null],["TR7NHqjeKQxGTCi8q8ZY4pL8otSzgjLj6t","USDT","Tether USD","TR7NHqjeKQxGTCi8q8ZY4pL8otSzgjLj6t",6,null,null,null],["TVj7RNVHy6thbM7BWdSe9G6gXwKhjhdNZS","KLV","Klever","TVj7RNVHy6thbM7BWdSe9G6gXwKhjhdNZS",6,null,null,null],["TZJmk51TP3YEETw7phRP581A7H3VzLDFsa","TEUR","EUR AlteWay Token","TZJmk51TP3YEETw7phRP581A7H3VzLDFsa",4,null,null,null]]
\ No newline at end of file
+[
+    ["TAFjULxiVgT4qWk6UZwjqwZXTSaGaqnVp4", "BTT", "BitTorrent", "TAFjULxiVgT4qWk6UZwjqwZXTSaGaqnVp4", 18, null, null, null],
+    ["TCFLL5dx5ZJdKnWuesXxi1VPwjLVmWZZy9", "JST", "JUST GOV", "TCFLL5dx5ZJdKnWuesXxi1VPwjLVmWZZy9", 18, null, null, null],
+    ["TLa2f6VPqDgRE67v1736s7bJ8Ray5wYjU7", "WIN", "WINK", "TLa2f6VPqDgRE67v1736s7bJ8Ray5wYjU7", 6, null, null, null],
+    ["TMwFHYXLJaRUPeW6421aqXL4ZEzPRFGkGT", "USDJ", "JUST Stablecoin", "TMwFHYXLJaRUPeW6421aqXL4ZEzPRFGkGT", 18, null, null, null],
+    ["TR7NHqjeKQxGTCi8q8ZY4pL8otSzgjLj6t", "USDT", "Tether USD", "TR7NHqjeKQxGTCi8q8ZY4pL8otSzgjLj6t", 6, null, null, null],
+    ["TVj7RNVHy6thbM7BWdSe9G6gXwKhjhdNZS", "KLV", "Klever", "TVj7RNVHy6thbM7BWdSe9G6gXwKhjhdNZS", 6, null, null, null],
+    ["TZJmk51TP3YEETw7phRP581A7H3VzLDFsa", "TEUR", "EUR AlteWay Token", "TZJmk51TP3YEETw7phRP581A7H3VzLDFsa", 4, null, null, null],
+    ["TAykmbrpL2VAemuRARXpTGjEttmsTjKPWA", "FIX00", "Fixed Asset Token", "TAykmbrpL2VAemuRARXpTGjEttmsTjKPWA", 2, null, null, null]
```

Comment:
@aelmilligy-ledger Could you please forward the info of this new token FIX00 to @adrienlacombe-ledger :pray: 
Thanks a lot